### PR TITLE
using ajax request instead of XMLHttpRequest

### PR DESCRIPTION
**Pull request by AlviseSembenico**

### CODE REVIEW

File: src/vng/testsession/api_views.py [hunk 1]
```diff
@@ -58,8 +58,6 @@ class SessionViewStatusSet(
         mixins.RetrieveModelMixin,
         viewsets.GenericViewSet):
     serializer_class = SessionStatusSerializer
-    authentication_classes = (TokenAuthentication, SessionAuthentication)
```
Comment (alextreme) [inline src/vng/testsession/api_views.py]: @AlviseSembenico I don't think you want to remove these, right?

Comment (AlviseSembenico) [reply src/vng/testsession/api_views.py]: Actually this API exposes only the deploying status therefore I don't see the need to protect it. Nonetheless I will do it

File: src/vng/testsession/api_views.py [hunk 2]
```diff
@@ -59,8 +59,6 @@ class SessionViewStatusSet(
         viewsets.GenericViewSet):
     serializer_class = SessionStatusSerializer
     authentication_classes = (TokenAuthentication, SessionAuthentication)
-    permission_classes = (permissions.IsAuthenticated, IsOwner)
```
Comment (alextreme) [inline src/vng/testsession/api_views.py]: @AlviseSembenico You also want to limit access to only the user that owns the session. There's no need to open this API further imho

Comment (AlviseSembenico) [reply src/vng/testsession/api_views.py]: @alextreme updated

**Pull request merged**